### DISCUSSION TITLE
Active development?

### DISCUSSION THREAD

**Discussion by lucai11** [PikaTimer/pikatimer#54] [General]
Hey @segfaultcoredump , just noticing some issues and recent responses from you indicating you're actively developing PikaTimer again? Would love to contribute to evolving this as it's an awesome piece of software but had assumed you'd stopped awhile back. Are you getting back into it?

**Comment by segfaultcoredump** [comment]
Don't call it a comeback, I've been here all along... :-)

Admittedly, development has been a bit, er, sporadic. I have a fork of the project over at https://github.com/segfaultcoredump/pikatimer that I push my development branch up to every now and then. I'm getting closer to adding a few things to finish out v1.6 and then kick off v1.7

But the bigger issue has been development time and wanting to work out the ecosystem a bit. As of right now, I have the following projects in flight or at least on the whiteboard:

- [PikaTimer](https://github.com/PikaTimer/pikatimer): The main scoring app
- [PikaReader](https://github.com/PikaTimer/PikaReader): An app designed to run on a small single board computer like a Raspberry and and sits between an RFID Reader like an Impinj R420 and PikaTimer or PikaRelay (Having PikaTimer directly communicate with a RFID Reader is asking for data loss.)
- PikaRelay: (Formerly [EchoServer](https://github.com/PikaTimer/EchoServer)) A php/mysql app that runs on a cloud hosted server so you can relay times from distant locations.  
- PikaReciever: A standalone app that lets you pull the data from a PikaReader based timing box or pull down data from PikaRelay and feed it into any other timing system that can accept a CSV or JSON based data feed. 
- PikaAnnouncer: (Formerly [Marmot](https://github.com/PikaTimer/Marmot)) This is an app that sits on the race announcer's laptop for a real time feed from PikaTimer

Many of them need some major overhaul as I migrate them from a Java 1.8 based system to Java 21. I also learned a lot over the past 8 years of development and thus have a lot of code cleanup to do. This is in addition to adding things like team scoring, lap based race support, and direct sync from various registration systems.

**Reply by lucai11** [reply]
Calling it a comeback! Any way to help you? Been a long time since I've used Java but would be keen to support if I could
